src/v/config/configuration.cc

```diff
@@ -39,24 +39,26 @@ configuration::configuration()
      .visibility = visibility::tunable},
     128_MiB,
     {.min = 1_MiB})
-  , log_segment_size_min(
```

Comment:
Storage team, I would appreciate it if you took a look at this commit since I'm changing configs that will affect you.
CC: @andijcr @andrwng

Reply:
A slack thread relevant to this https://redpandadata.slack.com/archives/C04AVDG7N58/p1695413863928209?thread_ts=1695411788.542979&cid=C04AVDG7N58